src/app/combo/combo.sample.ts

```diff
@@ -330,8 +332,13 @@ export class ComboSampleComponent implements OnInit, AfterViewInit {
         this.searchText = '';
     }
 
+    public handleSelectionChanging(evt: ISimpleComboSelectionChangingEventArgs)
     public handleSelectionChanging(evt: IComboSelectionChangingEventArgs) {
-        this.hasSelection = !!evt?.newSelection.length;
+        if (!evt.newSelection) {
+            this.simpleComboOpenOnClear.open();
+        } else {
+            this.hasSelection = !!evt?.newSelection.length;
```

Comment:
Actually, this needs to be called only if it is the combo that's throwing the `selectionChanging` event. So, instead of the overload we can have a union between `ISimpleComboSelectionChangingEventArgs` and `IComboSelectionChangingEventArgs` and then the condition can be put first and can be something like `if ('added' in evt) { /.../ return; }` - this will make sure that the event is of type `IComboSelectionChangingEventArgs` since out of the two only it has an `added` member. And it will also make sure that we don't call `this.simpleComboOpenOnClear.open` unless we explicitly want to.